styles/_fonts.scss

```diff
@@ -1,30 +1,30 @@
 @font-face {
-    font-family: "Apercu Pro";
-    src: url("/fonts/apercu/apercu-light-pro.woff2") format("woff2");
+    font-family: 'Apercu Pro';
+    src: url('/fonts/apercu/apercu-light-pro.woff2') format('woff2');
```

Comment:
Completely unrelated to this PR, please create a separate ticket – We don’t have the right to make those files available in an open-source project. If we want to use this font, then we’ll need to hotlink the files from `torchbox.com`.